src/app/components/Search/Instances.jsx

```diff
@@ -1,66 +1,50 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+import {
+  isEmpty as _isEmpty,
+} from 'underscore';
 
-class Instances extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.instances = props;
+export const Instances = (instances) => {
+  if (_isEmpty(instances.instances)) {
+    return null;
   }
 
-  parseInstances(instances) {
-    const items = instances.instances.map((instance, key) => (
+  const items = instances.instances.map((instance, key) => (
+    {
+      ebooks: (instance.items) ? instance.items : [],
+      pub_date: (instance.pub_date) ? parseInt(instance.pub_date) : null,
+      pub_place: (instance.pub_place) ? instance.pub_place : null,
+      publisher: (instance.publisher) ? instance.pub_publisher : null,
+      language: (instance.language) ? instance.language : null
+    }
+  ));
+
+  return (
+    <ul className="nypl-instances-list">
       {
-        ebooks: (instance.items) ? instance.items : [],
-        pub_date: (instance.pub_date) ? parseInt(instance.pub_date) : null,
-        pub_place: (instance.pub_place) ? instance.pub_place : null,
-        publisher: (instance.publisher) ? instance.pub_publisher : null,
-        language: (instance.language) ? instance.language : null
+        items.map((element, key) => (
+          <li className="nypl-results-item" key={key.toString()}>
+            <div className="nypl-results-description">
+              <p key={key.toString()}>
+                {
+                  element.ebooks.map((ebook, ebookKey) => (
```

Comment:
Will `element` always contain `ebooks`? Because this may not work if `element.ebooks` is undefined. A check to make sure it's defined would be helpful.

Reply:
But I guess you did default it to `[]` so that's good.